core/src/libtx/proof.rs

```diff
@@ -680,24 +632,9 @@ mod tests {
 			let commit = keychain.commit(amount, &id, &switch).unwrap();
 
 			// Generate proof with ProofBuilder..
-			let proof = create(
-				&keychain,
-				&builder,
-				amount,
-				&id,
-				&switch,
-				commit.clone(),
-				None,
-			)
-			.unwrap();
+			let proof = create(&keychain, &builder, amount, &id, &switch, commit, None).unwrap();
```

Comment:
Is rustfmt going to reformat these back to multiline fn calls (not sure when it decides to do this)?
Or was this actually the output of rustfmt?

Reply:
This is the output of rustfmt. I'm guessing that removing the `.clone()` gave us a line < MAX_CHAR_LINE or something like that.